### PR TITLE
Feature/checkout page

### PR DESCRIPTION
**Pull request by florencesitu**
- code cleaning
- updated color
- added placeholder
- updated delivery option

### CODE REVIEW

**Comment by 0804EshitaSharma** [comment]
Default branch is project_2, please merge that in your branch first.

File: client/src/components/Checkout/Checkout.css [hunk 1]
```diff
@@ -43,6 +44,10 @@
...
 
-/* @media only screen and (min-width : 1200px) {
-    .container { max-width: 1500px; }
-} */
\ No newline at end of file
+.radio-selection {
+    text-align: left;
+}
+
+input[type="radio"] {
```
Comment (0804EshitaSharma) [inline client/src/components/Checkout/Checkout.css:51]: Please don't use input instead use specific class names as it overrides all input tags this way.

**Comment by florencesitu** [comment]
I've updated the css and merge project2, please check if it's ok now

File: client/src/components/Checkout/Checkout.js [hunk 2]
```diff
@@ -1,264 +1,196 @@
...
+                                            />
+                                        </div>
+                                    </div>
+                                    <div className="col-md-12">
+                                        <div className="info-group mb-3">
+                                            <label>Card number</label>
+                                            <input
+                                                type="text"
+                                                className="info-input"
+                                                name="firstname"
```
Comment (ellee311) [inline client/src/components/Checkout/Checkout.js:154]: please change the names for this input and following to match the label

File: client/src/components/Checkout/Checkout.js [hunk 3]
```diff
@@ -1,264 +1,196 @@
...
+                                            />
+                                        </div>
+                                    </div>
+                                    <div className="col-md-12">
+                                        <div className="info-group mb-3">
+                                            <button type="button" className="btn">
```
Comment (ellee311) [inline client/src/components/Checkout/Checkout.js:180]: i don't think you need to specify that the type is button if you're using a button tag? unless this is a bootstrap thing, then ignore this comment

File: client/src/components/Checkout/Checkout.js [hunk 4]
```diff
@@ -1,264 +1,196 @@
...
+                                </span>
+                            </div>
+                            {selected1 && (
+                                <div className="wrapper">
+                                    <div className="row">
+                                        <LabeledInput
+                                            type="radio"
+                                            className="col-md-2"
+                                            name="delivery"
+                                            value="standard-delivery"
```
Comment (ellee311) [inline client/src/components/Checkout/Checkout.js:106]: i think you'll need to add in a value prop if you decide to use this with your LabeledInput component

File: client/src/components/Custom/LabeledInput.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+function LabeledInput({ className, name, label, type, placeholder }) {
+    return (
+        <div className={className}>
+            <div className="info-group mb-3">
+                <label>{label}</label>
+                <input
+                    name={name}
+                    type={type}
+                    className="info-input"
+                    placeholder={placeholder}
```
Comment (ellee311) [inline client/src/components/Custom/LabeledInput.js:12]: will have to do a check for if the placeholder is undefined in the case that a placeholder value is not passed into the component

Comment (florencesitu) [reply client/src/components/Custom/LabeledInput.js:12]: Hey Elsie, what do you mean by this? like if input is empty, give an error message or something?

Comment (ellee311) [reply client/src/components/Custom/LabeledInput.js:12]: yeah so I was wondering if a placeholder isn't passed in, will the labeled input component break? if it does, then there should probably be a check for that, but considering how close to deadline it is right now, maybe just write a todo note for yourself and then i can resolve this comment

Comment (florencesitu) [reply client/src/components/Custom/LabeledInput.js:12]: yea, I was looking into writing an error message for it, I'll see how much I can get done.

File: client/src/components/Checkout/DeliveryContainer.js [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+                        zipcode = component.long_name;
+                    }
+                    if (types.includes("country")) {
+                        country = component.long_name;
+                    }
+                    if (types.includes("administrative_area_level_1")) {
+                        province = component.long_name;
+                    }
+                }
+                setAddressComponents({ city, zipcode, country, province });
```
Comment (ellee311) [inline client/src/components/Checkout/DeliveryContainer.js:73]: need to switch the country and province around

**Review by ellee311 — APPROVED** [review]

**Pull request merged**